src/main/java/seedu/parser/CommandParser.java

```
@@ -131,10 +139,15 @@ public Command parseStoreResultsCommand(String text) {
             grade = split[0].trim().toUpperCase();
             gradeType = Module.checkGradeType(grade);
             moduleCode = split[1].trim().toUpperCase();
+            if (gradeType.equals("")) {
+                return new InvalidCommand();
+            } else {
+                return new StoreResultsCommand(grade, moduleCode, gradeType, isErrorThrown);
+            }
         } catch (UniModsException e) {
             System.out.println(e.getMessage());
+            return new InvalidCommand();
```

probably should throw exception rather than return invalidcommand